firmware/interpreter.cpp

```diff
@@ -0,0 +1,36 @@
+// Interpreter for serial commands
+
+#include <ArduinoJson.h>
+#include <StreamUtils.h>
+#include <SoftwareSerial.h>
+#include <string.h>
+#include "interpreter.h"
+#include "json.h"
+#include "sensor.h"
+
+int Interpreter::read(DynamicJsonDocument* doc){
+
+    if (Serial.available() > 0){
+        // read the incoming line:
+        str = "0";
```

Comment:
what's the purpose of this line?

Reply:
Oh yes you are correct. That line was copied over from some old code and I missed that line it should be removed.